spec/api/v1/time_ranges/index_spec.rb

```diff
@@ -22,7 +22,7 @@
       parameter name: 'page[number]', in: :query, type: :integer, required: false
       parameter name: 'filter[user_id]', in: :query, type: :integer, required: false
       parameter name: 'filter[time_range_type_id]', in: :query, type: :integer, required: false
-      parameter name: 'include', in: :query, type: :string, required: false
+      parameter name: 'include', in: :query, type: :string, required: false, example: 'tags'
```

Comment:
I like this addition of the example include, can we add it to spec/api/v1/time_ranges/get_spec.rb as well please